connector-packager/tests/test_resources/higher_min_version/manifest.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version='1.0' encoding='utf-8' ?>
```

Comment:
Feel free to make a highly simplified, but still valid, connector for test cases like this.  It doesn't actually need to run against anything.